2022-01-16 21:42:15,302 INFO    MainThread:4105 [wandb_setup.py:_flush():71] setting env: {}
2022-01-16 21:42:15,302 INFO    MainThread:4105 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-16 21:42:15,303 INFO    MainThread:4105 [wandb_init.py:_log_setup():371] Logging user logs to /home/labs/testing/class57/OCTransformer/breakdown/testings/vit_att_trial/wandb/run-20220116_214215-3thxg29s/logs/debug.log
2022-01-16 21:42:15,303 INFO    MainThread:4105 [wandb_init.py:_log_setup():372] Logging internal logs to /home/labs/testing/class57/OCTransformer/breakdown/testings/vit_att_trial/wandb/run-20220116_214215-3thxg29s/logs/debug-internal.log
2022-01-16 21:42:15,303 INFO    MainThread:4105 [wandb_init.py:init():404] calling init triggers
2022-01-16 21:42:15,303 INFO    MainThread:4105 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-16 21:42:15,303 INFO    MainThread:4105 [wandb_init.py:init():460] starting backend
2022-01-16 21:42:15,303 INFO    MainThread:4105 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-16 21:42:15,312 INFO    MainThread:4105 [backend.py:ensure_launched():216] starting backend process...
2022-01-16 21:42:15,317 INFO    MainThread:4105 [backend.py:ensure_launched():221] started backend process with pid: 4161
2022-01-16 21:42:15,317 INFO    MainThread:4105 [wandb_init.py:init():469] backend started and connected
2022-01-16 21:42:15,319 INFO    MainThread:4105 [wandb_init.py:init():533] updated telemetry
2022-01-16 21:42:15,331 INFO    MainThread:4105 [wandb_init.py:init():563] communicating current version
2022-01-16 21:42:16,103 INFO    MainThread:4105 [wandb_init.py:init():568] got version response 
2022-01-16 21:42:16,103 INFO    MainThread:4105 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-16 21:42:16,499 INFO    MainThread:4105 [wandb_init.py:init():606] starting run threads in backend
2022-01-16 21:42:18,287 INFO    MainThread:4105 [wandb_run.py:_console_start():1810] atexit reg
2022-01-16 21:42:18,288 INFO    MainThread:4105 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-16 21:42:18,288 INFO    MainThread:4105 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-16 21:42:18,291 INFO    MainThread:4105 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-16 21:42:18,291 INFO    MainThread:4105 [wandb_init.py:init():633] run started, returning control to user process
2022-01-16 21:42:22,647 INFO    MainThread:4105 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-16 21:42:22,649 INFO    MainThread:4105 [wandb_run.py:_restore():1752] restore
2022-01-16 21:42:24,987 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 812
}

2022-01-16 21:42:25,490 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 812
}

2022-01-16 21:42:25,592 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:25,693 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:25,795 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:25,896 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:25,998 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:26,100 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:26,203 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:26,304 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:26,406 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:26,507 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 812
  total_bytes: 2945
}

2022-01-16 21:42:26,609 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 914
  total_bytes: 2945
}

2022-01-16 21:42:26,711 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 914
  total_bytes: 2945
}

2022-01-16 21:42:26,812 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:26,914 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:27,015 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:27,117 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:27,218 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:27,320 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:27,555 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}

2022-01-16 21:42:28,608 INFO    MainThread:4105 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2945
  total_bytes: 2945
}
local_info {
}

2022-01-16 21:42:29,669 INFO    MainThread:4105 [wandb_run.py:_append_files():2180] logging synced files
